test/jdk/javax/swing/SwingWorker/TestSwingWorker.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/*
+ * @test
+ * @bug 8081474
+ * @summary  Verifies if SwingWorker calls 'done'
+ *           before the 'doInBackground' is finished
+ * @run main TestSwingWorker
+ */
+import javax.swing.SwingWorker;
+
+public class TestSwingWorker {
+
+    static boolean doInBackground =  false;
+
+    public static void main(String[] args) throws InterruptedException {
+        SwingWorker<String, String> worker =
+            new SwingWorker<String, String>() {
+            @Override
+            protected String doInBackground() throws Exception {
+                try {
+                    while (!Thread.currentThread().isInterrupted()) {
+                        System.out.println("Working...");
+                        Thread.sleep(1000);
+                    }
+                }
+                catch (InterruptedException ex) {
```

Comment:
```suggestion
                } catch (InterruptedException ex) {
```
`catch` should be on the same line as the closing `brace` of the previous block.